app/models/spree/order_decorator.rb

```diff
@@ -226,6 +226,10 @@ def set_variant_attributes(variant, attributes)
     end
   end
 
+  def shipment
+    shipments.first
+  end
+
```

Comment:
this is required for this test:
https://github.com/openfoodfoundation/openfoodnetwork/blob/2-0-stable/spec/models/spree/order_spec.rb#L711

Reply:
I searched for uses of `order.shipment` in the OFN code base and could only find specs. Is that right? In that case I would suggest to fix the spec instead of adding to the decorator.

Reply:
I have followed your advise. It's now done only in this spec and it works.